tests/sync/assessmentsSync.test.js

```diff
@@ -111,6 +111,50 @@ describe('Assessment syncing', () => {
     assert.equal(syncedData.assessment_questions[1].question.qid, util.ALTERNATIVE_QUESTION_ID);
   });
 
+  it('checks that shuffleQuestions defaults to true for exam-type assessments', async () => {
```

Comment:
Nit for consistency with other test names:
```suggestion
  it('defaults shuffleQuestions to true for an Exam-type assessment', async () => {
```